code/modules/overmap/ftl_shunt/core.dm

```diff
@@ -475,7 +476,13 @@ obj/machinery/ftl_shunt/core/proc/fuelpercentage()
 	desc = "A fuel port for an FTL shunt."
 	icon_state = "empty"
 
-	var/list/global/fuels = list(/decl/material/gas/hydrogen/tritium = 25000, /decl/material/gas/hydrogen/deuterium = 25000, /decl/material/gas/hydrogen = 25000, /decl/material/solid/exotic_matter = 50000)
+	var/list/global/fuels = list(
+				/decl/material/gas/hydrogen/tritium = 25000,
```

Comment:
You monkey's pawed me with your malicious compliance >:(
The `pretty` list format is one tab per entry.